suite-native/atoms/src/TitleHeader.tsx

```diff
@@ -0,0 +1,60 @@
+import { ReactNode } from 'react';
+
+import { TypographyStyle } from '@trezor/theme';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+
+import { Text } from './Text';
+import { VStack } from './Stack';
+
+type TitleHeaderProps = {
+    title?: ReactNode;
+    titleVariant?: TypographyStyle;
+    subtitle?: ReactNode;
+    textAlign?: 'left' | 'center';
+};
+
+const textStyle = prepareNativeStyle<{ textAlign: TitleHeaderProps['textAlign'] }>(
+    (_, { textAlign }) => ({
+        textAlign,
+        width: '100%',
```

Comment:
Why is the 100% width necessary? If it is applied to both texts, why not to set it for the VStack?

Reply:
I guees it breaks the layout on other places... 
![image](https://github.com/trezor/trezor-suite/assets/3729633/7cea2b53-a7b3-42b8-84c1-1dd299585eed)

Reply:
a16ef9a592